src/JsonApi/Serializer/ItemNormalizer.php

```diff
@@ -17,6 +17,7 @@
 use ApiPlatform\Api\ResourceClassResolverInterface;
 use ApiPlatform\Api\UrlGeneratorInterface;
 use ApiPlatform\Exception\ItemNotFoundException;
+use ApiPlatform\HttpCache\TagCollectorInterface;
```

Comment:
we want to avoid these kind of dependencies, the interface should be within our Serializer namespace, even if the implementation is the one from http cache.

Reply:
Got it. Will move to Serializer namespace.

Reply:
Done with https://github.com/api-platform/core/pull/5758/commits/720efb82c1ee1cded55a95b853e01a1c0b7c9e44